_pkgdown.yml

```diff
@@ -2,3 +2,16 @@ destination: docs
 template:
   params:
     bootswatch: lumen
+reference:
+  - title: "Manual layer"
+    desc: "These are manually written/curated functions which wrap some of the key calls into the OpenAPI autogen layer. They're more user-friendly and better documented."
+    contents:
+    - matches("^[a-z]")
+  - title: "Autogen layer"
+    desc: "These are functions autogenerated by the OpenAPI tool from the TileDB-Cloud API specification. They are thorough, but less user-friendly than the manual-layer functions."
+    contents:
+    - matches("^[^\\.a-z]")
+  - title: "Package-internal layer"
+    desc: "These are functions intended to be invoked by the manual-layer functions, as implementation details beneath the interest of most users of the API."
+    contents:
+    - matches("^\\.")
```

Comment:
How is this different from the preceding regexp in the sense of 'starts with a dot' and what follows is likely a letter too.

One the other hand you could hone in a `^manual` as a start.   (Maybe I am missing a step.  Apologies if I do.)

Reply:
I see -- and that may work well enough.  Upper-case is permitted, but "unusual".  We have some example as eg `Ecdf` by Harrell replacing / being an alternate for the baked-in `ecdf` from base (rather than rudely shadowing).  Using that for autogenerated code may actually be the best fit and lead to clean three-state partition.  Nice.